mpmcqueueof.go

```diff
@@ -1,16 +1,12 @@
-//go:build go1.19
-// +build go1.19
-
 package xsync
 
 import (
-	"runtime"
 	"sync/atomic"
 	"unsafe"
 )
 
 // A MPMCQueueOf is a bounded multi-producer multi-consumer concurrent
-// queue. It's a generic version of MPMCQueue.
+// queue.
```

Comment:
Same comment as above, seems like a good time to reclaim all these good type names :)